src/hooks.server.ts

```diff
@@ -0,0 +1,13 @@
+import type { Handle } from '@sveltejs/kit';
+import { sequence } from '@sveltejs/kit/hooks';
+// import Analytics from 'analytics-node';
+// import { env } from '$env/dynamic/private';
+
+// const analytics = new Analytics(env.ANALYTICS_WRITE_KEY || '');
+
```

Comment:
This may be the wrong place for calling the function, as its called during prerendering on the server at build time, and not at runtime. We need to have a look into an endpoint for this 🤔